core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -138,7 +138,7 @@ private[ui] class AllJobsPage(parent: JobsTab) extends WebUIPage("") {
               if (shouldShowCompletedJobs) {
                 <li>
                   <a href="#completed"><strong>Completed Jobs:</strong></a>
-                  {completedJobs.size}
+                  {listener.numCompletedJobs}
```

Comment:
would you like it to show both number in both the [summary at the top of the page](https://github.com/squito/spark/blob/SPARK-3454/core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala#L140) and also in the [heading above the table](https://github.com/squito/spark/blob/SPARK-3454/core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala#L162)??